mods/ra/rules/defaults.yaml

```diff
@@ -542,6 +545,7 @@
 	Armor:
 		Type: Light
 	UpdatesPlayerStatistics:
+		AddToArmyValue: true
```

Comment:
I admit I just wanted to re-use the `UpdatesPlayerStatistics` field already present here and didn't bother with spy planes and badger bombers because they cost $0 AFAIK. I fixed it in my latest push anyway.

WRT chinooks, I followed the "Army is everything that is not a building, and MCV or a harvester." rule, so they are included, as well as many other units. I think that's fine as is for now.